qa/TL3_EfficientDet_convergence/test_tensorflow.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+function CLEAN_AND_EXIT {
+    exit $1
+}
+
+cd /opt/dali/docs/examples/use_cases/tensorflow/efficientdet
+
+python -m pip install --upgrade pip
+python -m pip install -r requirements.txt
+
+
+python train.py \
+ --epochs 1 --multi_gpu --seed 1234 --pipeline dali_gpu --input coco \
+ --images_path /data/coco/coco-2017/coco2017/train2017 \
+ --annotations_path /data/coco/coco-2017/coco2017/annotations/instances_train2017.json \
+ --train_batch_size 4 --train_steps 4000 --eval_after_training --eval_during_training \
+ --eval_steps 1000 --eval_freq 1 2>&1 | tee $LOG
+
+
+CLEAN_AND_EXIT 0
```

Comment:
Do we need this? I though, that when the control flow reaches the end of file, the exit code is always 0?

Reply:
I cannot say for sure if we need this but I wrote it based on other tests and it's done in the same way in the YOLOv4 example so I think it is fine.